Pcap++/header/PcapFilter.h

```diff
@@ -266,7 +270,9 @@ namespace pcpp
 		 * @param[in] dir The address direction to filter (source or destination)
 		 * @param[in] ipv4Mask The mask to use. Mask should also be in a valid IPv4 format (i.e x.x.x.x), otherwise parsing this filter will fail
 		 */
-		IPFilter(const std::string& ipAddress, Direction dir, const std::string& ipv4Mask) : IFilterWithDirection(dir), m_Address(ipAddress), m_IPv4Mask(ipv4Mask), m_Len(0) {}
+		IPFilter(const std::string& ipAddress, Direction dir, const std::string& ipv4Mask) : IPFilter(IPv4Address(ipAddress), dir, ipv4Mask) {}
+
+		IPFilter(IPv4Address ipAddress, Direction dir, const std::string &ipv4Mask) : IFilterWithDirection(dir), m_Address(std::move(ipAddress)), m_IPv4Mask(ipv4Mask), m_Len(0) {}
```

Comment:
can we have `const IPv4Address&`? then you can do the copy constructor in the initialized list.

Reply:
Fixed 8c70753